build.gradle

```diff
@@ -148,8 +149,15 @@ def _numNodes = findProperty('numNodes') as Integer ?: 1
 
 def opensearch_tmp_dir = rootProject.file('build/private/opensearch_tmp').absoluteFile
 opensearch_tmp_dir.mkdirs()
-
+boolean isCiServer = System.getenv().containsKey("CI")
 test {
+    retry {
+        if (isCiServer) {
+            failOnPassedAfterRetry = false
+            maxRetries = 6
+            maxFailures = 10
```

Comment:
Haven't used test retry, just asking: if `maxRetries` is 6, the max failure should't exceed 7? Why set `maxFailures = 10`?